### DISCUSSION TITLE
Billing page says my preview has ended but I'm eligible to use it for free.

### DISCUSSION THREAD

**Discussion by crispyaus** [community/community#44433] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I noticed today my Co-pilot had stopped working on VSC, It stated an error in the corner saying my technical preview had ended. I have the student plan/feature on github, and it says I am eligible to use Co-pilot for free however there's no way to activate this or check that.

SS:
![image](https://user-images.githubusercontent.com/34707863/212597071-61ee9b9d-089f-4a5b-8f50-9ce8bd7d969b.png)

**Comment by iamcjwilliams** [comment]
Sorry for the trouble here, please visit [here](https://education.github.com/discount_requests/pack_application) in order to verify for free student access.
